SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -60,7 +60,7 @@ ComponentNamesAndIds ParseAvailableNamesAndIdsFromSchemaFile(const TArray<FStrin
 
 	for (const auto& SchemaLine : LoadedSchema)
 	{
-		FRegexPattern IdPattern(TEXT("(\tid = )([0-9]+)(;)"));
+		FRegexPattern IdPattern(TEXT("(id = )([0-9]+)(;)"));
```

Comment:
Are strict on white spaces in the rest of the statement? E.g should it be:
`"(id/s*=/s*)(/s*[0-9]+)(/s*;)"`

Reply:
Also not entirely sure why have three capture groups but this is turning into a digression that is probably out of scope.

Reply:
Yes, especially the trailing semi-colon capture group - but I won't tackle these issues in this PR